docs/create_organization_nailgun.py

```diff
@@ -5,9 +5,10 @@
 Use NailGun to accomplish this task.
 
 """
+from pprint import pprint
```

Comment:
Alphabetically this should be below nailgun. And looks like pre-commit doesn't cares much about alphabetically.

Are we going to change the rule of alphabetical order hence forth ?

Reply:
Imports are not sorted only alphabetically, but grouped into three distinct groups in order of: built-in imports, installed imports, and local imports.

In this case pprint is in the first group for built-in imports, there are no installed imports, and the nailgun imports are local to the project.

Reply:
Why I said this cause, we have been following the practice of sorting imports alphabetically. So now, We can just send an Email to the satqe list saying:
"Hey ! You need not to waste more time in sorting the imports, keep them as it is and pre-commit will take care"

Reply:
Exactly - its still sorting them alphabetically, just into groups as well.

Pre-commit should just modify the files automatically when a contributor makes a local commit, and they shouldn't have to worry about it (and reviewers don't have to worry or check the imports either!)

Reply:
To clarify, [Robottelo "code standards"](https://robottelo.readthedocs.io/en/latest/code_standards.html) mentions alphabetical sorting, but it also mentions grouping imports into three groups (standard library, third-party and application-specific). I believe most people understood it as alphabetical sorting within a group, not across the groups (because it doesn't make sense any other way).

`reorder-python-imports` uses the same rule. So it cleans up mess we have made, enforces following of rule for new code and (optionally) automatically applies it while developing.

Reply:
@mirekdlugosz Why I was more focusing on this because people were very much used to the way we were following earlier. And as I said earlier now they need not to worry about it.